acceptance/tests/apply/classes/parameterized_classes.rb

```diff
@@ -1,5 +1,9 @@
 test_name "parametrized classes"
 
+tag 'audit:high',
+    'audit:unit',  # This should be covered at the unit layer.
+    'audit:delete'
```

Comment:
This is a spurious use of the `audit:delete` tag. Thanks for pointing it out. I will remove it and the others as noted.